cmd/sgbouncewizard/main.go

```diff
@@ -55,12 +136,125 @@ func ListRules(w http.ResponseWriter, r *http.Request) {
 
 func generateHash(pwd []byte) string {
 	hash, err := bcrypt.GenerateFromPassword(pwd, bcrypt.MinCost)
+	if err != nil {
+		log.Println(err)
+		return "error"
```

Comment:
We should bubble the error up, and let the calling function decide how to handle the error. Consider revising the signature to `func generateHash(pwd []byte) (string, error)` to return an the actual error, instead of a  unique "error" string.